smart_simulation/Notebooks/modeling.ipynb

```diff
@@ -0,0 +1,942 @@
+{
```

Comment:
<p><code>Currently we're trying to answer question (2). </code> typo? Is it 3 or 4 instead?</p><p> </p><p><code>calculate consumption from historic weight</code>  I'd add "from historic weight changes" to clarify.</p><p> </p><p><code>consumer might self-ration, but unlikely, or infrequent</code>  can you clarify what this means?</p><p> </p><p>I think you need to be more explicit about what you call "weight based forecast", since what you <em>actually</em> have access to is the historic weight measurements, not consumption. I think what you're getting at is that weight is a combined relationship between the consumer and the supplier, while consumption is only dependent on the consumer. Thus its easier to model just the consumer's behavior independent of the supplier behavior. Is this a fair way of putting it?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbpauly/smart-simulation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Yes, you're correct. The way it reads, it's question 4, however that's a formatting issue. Thanks for the other comments too. I will make the changes.</p>